Makefile

```diff
@@ -236,8 +237,8 @@ CXXFLAGS ?=
 LIB_FUZZING_ENGINE ?= -libfuzzer
 OUT ?= .
 
-$(OUT)/vm-fuzzer: $(GOFILES) | $(GOFUZZBUILD)
-	$(GOFUZZBUILD) -libfuzzer -o fuzzer.a ./internal/vm
+$(OUT)/vm-fuzzer: $(GOFILES) | $(GOFUZZ)
+	$(GOFUZZ) -o fuzzer.a github.com/google/mtail/internal/vm
 	$(CXX) $(CXXFLAGS) $(LIB_FUZZING_ENGINE) fuzzer.a -lpthread -o $(OUT)/vm-fuzzer
```

Comment:
With the change above, you won't need to rename GOFUZZBUILD to GOFUZZ.

I am confused why this PR doesn't remove the CXX line though, if this change uses "native go-fuzz"; I don't see where go-fuzz is being invoked.